### ISSUE TITLE
Autogenerate config with subfolders with common import files

### ISSUE THREAD

**Issue by mariomurrent-softwaresolutions** [dushanpantic/grpc-server-mock#1]
Is there a possibility to exclude special folders from autogeneration?
This would be helpful, if there are some folders for common .proto files.

**Comment by dushanpantic** [comment]
I don't understand the question. Can you explain it in more detail?

**Comment by mariomurrent-softwaresolutions** [comment]
E.g: There is a common folder where common .proto files are located. These files are used in different .proto files as import.
Currently for every subfolder in the autowired example a service is beeing generated - but for the common folder there should not be generated a new service, because the are just common files which are imported in other files.

**Comment by dushanpantic** [comment]
Services are being generated only when a folder with their responses is present.
Do you have an example repo of the behavior you described?

**Comment by mariomurrent-softwaresolutions** [comment]
I will provide you with an example

**Comment by mariomurrent-softwaresolutions** [comment]
@dushanpantic I added an example folder in the PR #3.
If you try the auto wire command in this folder it won't work

**Comment by dushanpantic** [comment]
Thanks, I will check it out in the evening when I get back home.

**Comment by mariomurrent-softwaresolutions** [comment]
Alright, thx - and no stress. It's actually nice working with you on this topic :) Thx for your time and interest

**Comment by dushanpantic** [comment]
Likewise!

**Comment by dushanpantic** [comment]
I added the requested feature in `0.0.11` (https://github.com/dushanpantic/grpc-server-mock/commit/34adc13b80c3459dc83e806a37c6c208c1126ea3).

In the example you provided, create `root.json` on path: `examples/autowire-complex/root.json` and populate it with:
```
{
  "ignoreServiceFolders": ["common"]
}
```
Just bear in mind that there is an error in the provided `.proto` file, and you need to edit it from this:
```
...

message SearchRequest {
  string query = 1;
  int32 page_number = 2;
  int32 result_per_page = 3;
  SearchType subscription_type = 2;
}

...
```
to this:
```
...

message SearchRequest {
  string query = 1;
  int32 page_number = 2;
  int32 result_per_page = 3;
  SearchType subscription_type = 4;
}

... 
```

**Comment by mariomurrent-softwaresolutions** [comment]
Thx, yeah that error was a copy & paste error, but it did not work anyway.
Thx for the fix

**Comment by mariomurrent-softwaresolutions** [comment]
Fixed with #3
